dali/pipeline/operators/reader/parser/parser_test.cc

```diff
@@ -82,6 +82,8 @@ TYPED_TEST(ParserTest, BasicTest) {
   IntArrayParser<CPUBackend> parser(OpSpec("temp"));
   IntArrayWrapper ia_wrapper = {data, 3 + H*W*C};
   parser.Parse(ia_wrapper, ws);
+  delete [] data;
```

Comment:
if possible use containers or smart pointers for those

Reply:
Done